lib/ReactViews/Workbench/WorkbenchItem.jsx

```diff
@@ -2,14 +2,18 @@
 
 import classNames from "classnames";
 import createReactClass from "create-react-class";
+import { runInAction } from "mobx";
 import { observer } from "mobx-react";
 import PropTypes from "prop-types";
 import React from "react";
 import { sortable } from "react-anything-sortable";
+import { withTranslation } from "react-i18next";
 import defined from "terriajs-cesium/Source/Core/defined";
+import getPath from "../../Core/getPath";
 import CommonStrata from "../../Models/CommonStrata";
-import getAncestors from "../../Models/getAncestors";
+import Box from "../../Styled/Box";
 import Icon from "../Icon";
+import PrivateIndicator from "../PrivateIndicator/PrivateIndicator";
```

Comment:
How is this sorted? # of traversals?

Reply:
It looks like alphabetically with less local modules first. It's vscode's default when you run `Organise imports`. It's a de facto standard that Kevin and I have used for a couple of years. Just a little neater than having everything everywhere.

Reply:
It might make merges a bit messy, but in a 3 way merge you can accept both current changes and incoming changes, then run `Organise imports` again and it'll remove all duplicate and unused imports and order them again.